coalib/results/HiddenResult.py

```diff
@@ -39,8 +39,8 @@ def __init__(self,
         :param origin:
             The originating bear.
         :param contents:
-            Any object that is picklable since it will be
-            transferred across processes.
+            Used to send additional data (arbitrary python objects) besides
```

Comment:
Please make it "Any object to send additional data besides....."
would be better this way. :)

Reply:
Right. That does seem more appropriate. Will make the required changes :+1: